bin/move_it_mirror.py

```diff
@@ -103,8 +105,16 @@ class Listeners(object):
 
     def __init__(self, callback, client_topic, providers, **attrs):
         self.listeners = []
+        if client_topic is None:
+            topics = []
+        else:
+            topics = [client_topic]
         for provider in providers.split():
-            self.listeners.append(Listener('tcp://' + provider, [client_topic],
+            if '/' in provider:
+                parts = provider.split('/', 1)
+                provider = parts[0]
+                topics.append('/' + parts[1])
```

Comment:
So one provider can have multiple topics ?

Reply:
The underlying `NSSubscriber` used by the client uses list of topics, so thought that would be the way to go. Do you think the `provider` topic, if specified, should overwrite the global topic?

Reply:
yes, otherwise it might lead to unwanted topics being monitored